components/onboarding/onboardingService.js

```diff
@@ -0,0 +1,15 @@
+import axios from '../axios'
+import fs from 'fs'
+import path from 'path'
+
+const onboardingTextPath = path.join(process.cwd(), 'config', 'onboarding.txt')
+const text = fs.readFileSync(onboardingTextPath).toString()
```

Comment:
You could consider using an asynchronous method instead of the asynchronous method to read files. Could you verify this situation in this blog post: https://adrianmejia.com/asynchronous-vs-synchronous-handling-concurrency-in-javascript/

Reply:
@rtarantelli, this case is a single time reading only and store the text on memory, and will be called on application boot only, there is no problem on reading the file syncrhonous here. To change this behavior to async, the method sendOnboardingMessage should read the file asynchronously  on every call. Agreed?